internal/bgp/frr/frr_test.go

```diff
@@ -878,12 +889,19 @@ func TestTwoAdvertisementsTwoSessions(t *testing.T) {
 		Communities: communities,
 		LocalPref:   2,
 	}
-
-	err = session.Set(adv1, adv2)
+	advs := []*bgp.Advertisement{adv1, adv2}
+	rand.Shuffle(len(advs), func(i, j int) {
+		advs[i], advs[j] = advs[j], advs[i]
+	})
+	err = session.Set(advs[0], advs[1])
 	if err != nil {
 		t.Fatalf("Could not advertise prefix: %s", err)
 	}
-	err = session1.Set(adv1, adv2)
+
+	rand.Shuffle(len(advs), func(i, j int) {
```

Comment:
Can we add more attempts to shuffle and check like in `config_conversion_test.go` so we'll be more confident?

Reply:
This will require a bit of reworking, as I don't want to pay the FRR extra effort every iteration. Let me try to fix it.